src/autolabel/transforms/pdf.py

```diff
@@ -2,19 +2,21 @@
 
 from autolabel.schema import TransformType
 from autolabel.transforms import BaseTransform
+from autolabel.cache import BaseCache
 
 
 class PDFTransform(BaseTransform):
     def __init__(
         self,
+        cache: BaseCache,
```

Comment:
in the BaseTransform init `cache` is the last param, we should probably maintain this for other downstream transforms as well - not a requirement but for readability